README.md

```diff
@@ -100,3 +100,5 @@ Creating a source or destination connector will fail in the next cases:
 
 ## Useful resources
 * [Quotas and limits in Amazon Redshift](https://docs.aws.amazon.com/redshift/latest/mgmt/amazon-redshift-limits.html)
+
+![scarf pixel](https://static.scarf.sh/a.png?x-pxid=617ebf4a-148c-44ad-8f64-6cc5780d34ae)
```

Comment:
I don't think we have a pixel dedicated to the redshift connector, I'll create one so we can replace it here

Reply:
here we go :D
```suggestion
![scarf pixel](https://static.scarf.sh/a.png?x-pxid=dc0e518d-385c-4e33-bd1c-6b4d2eaebb74)
```

Reply:
Updated. Thank you!